services/ext/api.go

```diff
@@ -1484,6 +1484,10 @@ func (api *PublicAPI) ListenAddresses() ([]string, error) {
 	return api.service.messenger.ListenAddresses()
 }
 
+func (api *PublicAPI) Enr() (string, error) {
```

Comment:
I intentionally use `Enr` because if i used `ENR` the RPC method would look like this: `wakuext_eNR`